src/main/java/gameframework/gui/GameWindow.java

```diff
@@ -32,11 +62,39 @@ public GameWindow(String gameName, GameCanvas gameCanvas,
 		this.statusBar.addAll(elementsStatusBar);
 		this.frame = new Frame(gameName);
 		this.gameCanvas = gameCanvas;
-		this.gameCanvas.setSize(//
```

Comment:
please don't touch lines if you don't have anything to change. Keep the diff as simple as possible to ease code review